migrations/20231210123554_init.sql

```diff
@@ -0,0 +1,236 @@
+-- Enums
+
+CREATE TYPE "Gender"              AS ENUM ('male', 'female');
+CREATE TYPE "Role"                AS ENUM ('user', 'admin');
+CREATE TYPE "Status"              AS ENUM ('available', 'unavailable');
+CREATE TYPE "Association"         AS ENUM ('sponsor', 'organization', 'media');
+CREATE TYPE "TaskPriority"        AS ENUM ('low', 'medium', 'high');
+CREATE TYPE "AcceptanceStatus"    AS ENUM ('pending', 'accepted', 'rejected');
+CREATE TYPE "EmployeeLevel"       AS ENUM ('basic', 'organizer', 'manager');
+CREATE TYPE "EmployeeContract"    AS ENUM ('full-time', 'part-time', 'temporary');
+CREATE TYPE "EventRole"           AS ENUM ('worker', 'staff');
+
+-- Tables
+
+CREATE TABLE IF NOT EXISTS "User"
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    name        VARCHAR(255) NOT NULL,
+    email       VARCHAR(255) NOT NULL UNIQUE,
+    birth       DATE NOT NULL,
+    avatar_url  VARCHAR(255),
+    gender      "Gender" NOT NULL,
+    role        "Role" NOT NULL,
+    status      "Status" NOT NULL,
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP
+);
+
+
+CREATE TABLE IF NOT EXISTS "Company"
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    name        VARCHAR(255) NOT NULL,
+    description TEXT,
+    website     VARCHAR(255),
+    crn         VARCHAR(16) NOT NULL UNIQUE,
+    vatin       VARCHAR(18) NOT NULL UNIQUE,
+    phone       VARCHAR(255) NOT NULL UNIQUE,
+    email       VARCHAR(255) NOT NULL UNIQUE,
+    avatar_url  VARCHAR(255),
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP
+);
+
+
+CREATE TABLE IF NOT EXISTS "Event"
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    name        VARCHAR(255) NOT NULL,
+    description TEXT,
+    website     VARCHAR(255),
+    with_staff  BOOLEAN NOT NULL,
```

Comment:
This naming doesn't make much sense.
This bool should track if the staff accepts new staff, not that it 'has staff' (which the name suggests)